controllers/topics.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { selectTopics } = require("../models/topics.models")
+
+module.exports = {
+
+  getTopics: function(req, res, next) {
+    selectTopics()
+      .then(topics => {
+        res.status(200).send({ topics });
+      })
```

Comment:
Hey Pijus, here we should be adding a catch block in order to deal with any errors.

Reply:
Please also ensure you handle any 404 or 500 error :)